docs/source/settingup.rst

```diff
@@ -151,6 +151,42 @@ Once the dataset is available on Dataverse, it can also be cloned using the
       'datalad-annex::?type=external&externaltype=dataverse&encryption=none&exporttree=no&url=https%3A//demo.dataverse.org&doi=doi:10.70122/MYT/ESTDOI'
 
 
+Help others getting started
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Its not always obvious to outsiders whether a Dataverse Dataset is a DataLad Dataset as well.
+To help others get started with your shared datasets, we recommend to add a short DataLad-specific description to your dataset's "Description" metadata.
+For convenience, a template to copy-and-paste can be found below.
+
+.. code-block:: text
+
+   This dataset is a <b>DataLad dataset</b> (https://www.datalad.org), published with
+   the <code>DataLad-dataverse</code> software (https://docs.datalad.org/projects/dataverse).
+   If you <code>"datalad clone"</code> it, it provides fine-grained data access down to the
+   level of individual files, and allows for tracking future updates. For this,
+   DataLad and datalad-dataverse are required. You can find installation instructions
+   at https://docs.datalad.org/projects/dataverse/settingup.html#installation.
```

Comment:
```suggestion
   at https://docs.datalad.org/projects/dataverse/en/latest/settingup.html#installation.
```
The original URL 404s for me, this one doesn't.

Reply:
Thanks for catching that! Rather than this bulky URL, I'd slightly prefer to add this redirect to readthedocs, though. I think its a pretty common thing to be able to omit "/lang/version/"
(will take a look after I finished another thing)

Reply:
The shorter URL should work now :)